src/vignette/http/route_helpers.clj

```diff
@@ -28,6 +31,25 @@
     (create-head-response image-params)
     (error-response 404 image-params)))
 
+(defn error-catcher [request handler]
+  (try+
+    (apply handler [])
+    (catch Exception e
+      (println (.getMessage e) ":" (:uri request))
+      (perf/publish {:exception-count 1})
+      (log/warn (str e) {:path  (:uri request)
```

Comment:
do we need both `println` and `log`?

Reply:
println goes to consul output only, where log goes to kibana and is not outputted to consol on default